src/controller/data_model/controller-clusters.matter

```diff
@@ -478,15 +478,15 @@ client cluster AccessControl = 31 {
     nullable INT16U adminPasscodeID = 2;
     ChangeTypeEnum changeType = 3;
     nullable AccessControlEntry latestValue = 4;
-    fabric_idx adminFabricIndex = 254;
+    fabric_idx fabricIndex = 254;
```

Comment:
Well, the field has to exist in the end, right?  Spec says all fabric-scoped structs have it.

We used to have it in the XML.  This PR takes it out of the XML, but still injects it into the ZAP database, so that anything querying that database (like "give me the fields in this struct") will know it's there and act accordingly, since the field _does_ exist in the struct.  The .matter file is generated from the database, so it has that field.

We _could_ take it out of the .matter file too, if we wanted to.  We'd then need some other IDL annotation for "this is a fabric-scoped struct" and anything that consumes the IDL would need to know to add the field it as needed.